<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>YinYue</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QSplitter" name="splitter">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <widget class="QWidget" name="leftWidget" native="true">
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QListView" name="playlistView"/>
        </item>
        <item>
         <layout class="QHBoxLayout" name="playlistControlLayout">
          <item>
           <widget class="QPushButton" name="addButton">
            <property name="text">
             <string>添加</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="removeButton">
            <property name="text">
             <string>移除</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="clearButton">
            <property name="text">
             <string>清空</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="rightWidget" native="true">
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <widget class="QLabel" name="currentSongLabel">
          <property name="text">
           <string>No song playing</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="timeLayout">
          <item>
           <widget class="QLabel" name="currentTimeLabel">
            <property name="text">
             <string>00:00</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QSlider" name="positionSlider">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="totalTimeLabel">
            <property name="text">
             <string>00:00</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="controlLayout">
          <item>
           <widget class="QPushButton" name="previousButton">
            <property name="text">
             <string/>
            </property>
            <property name="icon">
             <iconset theme="media-skip-backward"/>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="playButton">
            <property name="text">
             <string/>
            </property>
            <property name="icon">
             <iconset theme="media-playback-start"/>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="stopButton">
            <property name="text">
             <string/>
            </property>
            <property name="icon">
             <iconset theme="media-playback-stop"/>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="nextButton">
            <property name="text">
             <string/>
            </property>
            <property name="icon">
             <iconset theme="media-skip-forward"/>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="volumeLayout">
          <item>
           <widget class="QLabel" name="volumeLabel">
            <property name="text">
             <string>音量</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QSlider" name="volumeSlider">
            <property name="maximum">
             <number>100</number>
            </property>
            <property name="value">
             <number>50</number>
            </property>
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="toolLayout">
          <item>
           <widget class="QPushButton" name="playModeButton">
            <property name="text">
             <string>顺序播放</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="dlnaButton">
            <property name="text">
             <string>DLNA</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>30</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
